proposals/lip-0051.md

```diff
@@ -511,6 +512,9 @@ def verify(trs: Transaction) -> None:
     if receivingChainID == OWN_CHAIN_ID:
         raise Exception("Receiving chain cannot be the sending chain.")
 
+    if not Interoperability.isChannelActive(receivingChainID):
```

Comment:
The previous check is superfluous now

Reply:
As discussed, it's not wrong to keep this check for reasons of clarity.